modules/tinymce/src/core/main/ts/fmt/ApplyFormat.ts

```diff
@@ -269,33 +265,18 @@ const applyFormat = (ed: Editor, name: string, vars?: FormatVars, node?: Node |
         return count;
       };
 
-      const getChildElementNode = (root: Node): Node | false => {
-        let child: Node | false = false;
-        each(root.childNodes, (node) => {
-          if (isElementNode(node)) {
-            child = node;
-            return false; // break loop
-          }
-        });
-        return child;
-      };
-
-      const mergeStyles = (node: Element) => {
-        let clone;
-
-        const child = getChildElementNode(node);
-
-        // If child was found and of the same type as the current node
-        if (child && !Bookmarks.isBookmarkNode(child) && MatchFormat.matchName(dom, child, format)) {
```

Comment:
I intentionally removed the `!Bookmarks.isBookmarkNode(child)` check here as it's done in `isElementNode` already, so this was just a duplicate check.